gpc-api-facade/src/main/resources/db/changelog/migration/01-create-patient_migration_request.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE patient_migration_request
+(
+    id character varying(36) NOT NULL
```

Comment:
2 comments here:
- maybe we should use liquibase syntax - to abstract from sepecific database?
- can we create sequence for ID column? and actually make it PK?